accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/impl/AcInstallationServiceImpl.java

```diff
@@ -144,6 +146,9 @@
         
         @AttributeDefinition(name="Configuration path(s)", description="JCR path(s) where the config files reside (usually it's just one, can be multiple for multitenant setups)")
         String[] configurationRootPaths() default {};
+        
+        @AttributeDefinition(name="Allowed principals for unmanaged path(s)", description="List of principals for which the unmanaged paths should be checked")
+        String[] allowedPrincipalsForUnmanagedPaths() default {};
```

Comment:
Introducing a new config property is something we would want to avoid (also if we introduced one, it should be part of the global config section, the strategy is to have `configurationRootPaths` as the only OSGi config prop, `intermediateSaves` has been removed in the latest version since 3.x

Reply:
ok